test/libsolidity/syntaxTests/denominations/denominations.sol

```diff
@@ -0,0 +1,10 @@
+contract C {
```

Comment:
Is this a new test? If yes, please remove the error. If no, please remove the original :)

Reply:
Added the error on purpose to have more things to check for. You want it to be in a separate file?

Reply:
Yes, I think that would be better.

Reply:
I've split it.